pkgs/development/tools/misc/ccls/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
     # We need to tell ccls where to find the standard library headers.
 
     standard_library_includes="\\\"-isystem\\\", \\\"${lib.getDev stdenv.cc.libc}/include\\\""
-    standard_library_includes+=", \\\"-isystem\\\", \\\"${llvmPackages.libcxx}/include/c++/v1\\\""
+    standard_library_includes+=", \\\"-isystem\\\", \\\"${lib.getDev llvmPackages.libcxx}/include/c++/v1\\\""
```

Comment:
I think this approach would be more correct: https://github.com/NixOS/nixpkgs/pull/120229/files

Reply:
Your comment on the other PR is that you don't like it and would like to see something more precise, so I'm not sure what you're suggesting.  Should we read the compiler flags from `${clang}/nix-support/libcxx-cxxflags` instead?

Reply:
Ok, I've changed the wrapper to read the libc-cflags and libcxx-cxxflags instead.